app/components/address-search.hbs

```diff
@@ -1,65 +1,48 @@
 {{#let @itemComponent (unique-id) as |Item id|}}
-  {{#if this.isAddressSearchMode}}
-    <Item
-      @labelFor="{{id}}-address-search"
-      @required={{@required}}
-      @errorMessage={{if @error "Selecteer een adres"}}
-      @alignTop={{true}}
-    >
-      <:label>Adres</:label>
-      <:content as |errorMessage|>
-        <AddressRegisterSelector
-          @id="{{id}}-address-search"
-          @address={{@address}}
-          @onChange={{this.handleAddressChange}}
-          @error={{errorMessage}}
-        />
-        {{#if errorMessage}}
-          <AuHelpText @error={{true}}>{{errorMessage}}</AuHelpText>
-        {{/if}}
-        <p class="au-u-margin-top-tiny">
-          <AuButton @skin="link" {{on "click" this.toggleInputMode}}>
-            Vul adres manueel in
-          </AuButton>
-        </p>
-      </:content>
-      <:error>{{! Disable the default error message}}</:error>
-    </Item>
-    <Item
-      @labelFor="{{id}}-address-search-bus-select"
-      @required={{not this.isBusNumberSelectionDisabled}}
-    >
-      <:label>Busnummer</:label>
-      <:content>
-        <AddressRegisterBusNumberSelector
-          @id="{{id}}-address-search-bus-select"
-          @address={{this.addressWithBusNumber}}
-          @options={{this.addressesWithBusNumbers}}
-          @onChange={{this.handleBusNumberChange}}
-          @disabled={{this.isBusNumberSelectionDisabled}}
-          @placeholder={{if
-            this.showBusNumbersNotAvailableMessage
-            "Geen busnummer beschikbaar"
-          }}
-        />
+{{#if this.isAddressSearchMode}}
+<Item @labelFor="{{id}}-address-search" @required={{@required}} @errorMessage={{if @error "Selecteer een adres" }}
```

Comment:
Could you please disable the auto-formatter in your editor or make sure it uses Prettier :smile:. It seems to be adding a lot of changes that are unrelated to the work.